internal/website/sidebars.js

```diff
@@ -1,47 +1,73 @@
 module.exports = {
-  docs: {
-    Introduction: [
-      'introduction/getting-started',
-      'introduction/installation',
-      'introduction/features',
-      'introduction/motivation',
-      'introduction/new-in-2',
-    ],
-    CLI: ['cli/config', 'cli/codegen', 'cli/javascript', 'cli/programmatic'],
-    React: [
-      'react/basic-usage',
-      'react/caveats',
-      'react/config',
-      'react/query',
-      'react/mutations',
-      'react/subscriptions',
-      'react/suspense',
-      'react/meta',
-      'react/ssr',
-      'react/api-reference',
-      {
-        type: 'category',
-        label: 'Auto-generated API Reference',
-        items: require('./sidebars/react.js'),
-      },
-    ],
-    Core: [
-      'core/basic-usage',
-      'core/helpers',
-      'core/config',
-      'core/mutations',
-      'core/subscriptions',
-      'core/persistence',
-      'core/graphql-upload',
-      'core/api-reference',
-      {
-        type: 'category',
-        label: 'Auto-generated API Reference',
-        items: require('./sidebars/core.js'),
-      },
-    ],
-    'Development Tools': ['development/logger'],
-    FAQ: ['faq'],
-    Contributing: ['contributing'],
-  },
+  docs: [
+    {
+      type: 'category',
+      label: 'Introduction',
+      collapsed: false,
+      items: [
+        'intro',
+        'intro/how-it-works',
+        // 'introduction/features',
```

Comment:
why are you removing the features page from the index but keeping the doc itself?

Reply:
going to update homepage later